Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java

```diff
@@ -55,11 +58,16 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
 
+
+    dataBaseOpenHelper = new DataBaseOpenHelper(this);
+
+
     Button btnFeedlyAccount = (Button) findViewById(R.id.FeedlyAccountBtn);
     btnFeedlyAccount.setOnClickListener(new Button.OnClickListener() {
       public void onClick(View v) {
         final String URL = "https://cloud.feedly.com/v3/streams/contents?streamId=user/" + ID + "/category/global.all";
-        new GetPageList().execute(URL);
+        new GetPageList(dataBaseOpenHelper).execute(URL);
```

Comment:
In that case, I'm worrying whether this `dataBaseOpenHelper` is thread safe or the calling routine for this `onClick` can guarantee to be run on a single thread, in case the user calls this event before the first one ends. If this part is clear, I'm okay with the current code. Otherwise it would be better to make Helper on every `onClick`.